src/communication/serverMessages.ts

```diff
@@ -25,16 +26,18 @@ export interface Data {
 
 export interface CommandData {
     Command: string;
-    JsonMessage: string;
+    Data: any;
```

Comment:
You should try to create a type for Data.
Is it different from the Data type used in this file ?